### ISSUE TITLE
so many new issues since update

### ISSUE THREAD

**Issue by meksrc** [ianyh/Amethyst#1477]
**Describe the bug**
Pretty much all the time, sometimes out of nowhere, sometimes when i wake my laptop up, everything starts to rearrange.
Windows get thrown to different spaces, some spaces are complety broken, windows overlap, get resized, go floating, all of it.
I have to close the space then and create a new one, this seems to work.
Can i downgrade again? i did not have so many issues on the last versions.
But still, great app! I hope it gets better :) I dont really want to switch to yabai.

**Applications:**
all of them, safari, vscode, notion, discord, etc.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Versions:**
 - macOS: latest
 - Amethyst: latest
 
**Debug Info**
```
$ /Applications/Amethyst.app/Contents/MacOS/Amethyst --debug-info [--include-apps]
```
Note: `--include-apps` will list your manageable applications, but is optional if you don't want to list that.

**Additional context**

**Comment by ianyh** [comment]
Yikes! Does latest mean 0.19 or 0.20b?

**Comment by meksrc** [comment]
Sorry, its 0.19 :)

**Comment by sat-insitro** [comment]
Yup, I'm also getting issues with this update, specifically clicking windows seem to rearrange them. Any advice?

**Comment by memco** [comment]
I have also had these same issues. Is there any information needed to help diagnose the problem?

**Comment by ianyh** [comment]
Trying to go through backlog. Is this still happening?

**Comment by memco** [comment]
No. I don't have this problem anymore on 0.24.3. (and probably several prior versions).
